docs/networking/networking.md

```diff
@@ -136,7 +136,12 @@ By renaming/updating interfaces like this, you can assure all your hosts have th
 
 ### Remove a physical NIC
 
-Before removing it, just be sure to remove its associated networks, so it won't cause trouble. Then, shutdown, remove the NIC and finally boot. After the boot, do a `xe pif-forget uuid=<OLD PIF UUID>` to get rid of the object record.
+Before removing a physical NIC, ensure that no VMs are using the interface. Shutdown the host, physically remove the NIC and boot.
+After boot, the PIF will need to be removed.
```

Comment:
Ideally:
"After boot, the PIF will need to be removed. You can do it this way:"

Reply:
I also wonder if this can be done via Xen Orchestra directly? In the host/network view? Can you check?

Reply:
Made requested change. I will explore if this step can be done with XO. Although might not be for a day or 2 based on personal schedule.

Reply:
No worries, let's merge this first and you could do another PR with some XO screenshots later. Thanks a lot!